hack/ci/testdata/seed.yaml

```diff
@@ -159,6 +159,24 @@ spec:
             minimumMemory: 0
             minimumVCPUs: 0
           region: dbl
+    syseleven-fes1:
+      country: DE
+      location: Syseleven - fes1
+      spec:
+        openstack:
+          authURL: https://api.cbk.cloud.syseleven.net:5000/v3
+          availabilityZone: fes1
+          dnsServers:
+            - 37.123.105.116
+            - 37.123.105.117
+          enforceFloatingIP: true
+          images:
+            centos: "CentOS 8 (2021-07-05)"
+            ubuntu: "Ubuntu Focal 20.04 (2021-07-01)"
```

Comment:
Why did we change the availability zone and images?

Reply:
IPv6 is available only in this zone, and images are from the ones that are available in that zone